daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/SBuiltinTest.scala

```diff
@@ -1413,7 +1415,18 @@ object SBuiltinTest {
   }
 
   private def evalSExpr(e: SExpr, onLedger: Boolean): Either[SError, SValue] = {
-    val machine = Speedy.Machine.fromPureSExpr(compiledPackages, e, onLedger)
+    val machine = if (onLedger) {
```

Comment:
This seems a bit messy. Not quite sure what would be better. Maybe just a helper method to evaluate `pure x` in a scenario context?

Reply:
I think it is OK for tests.